### ISSUE TITLE
MD025: front_matter_title: "" does not disable title checking

### ISSUE THREAD

**Issue by ianlewis** [DavidAnson/markdownlint#1493]
According to the [docs](https://github.com/DavidAnson/markdownlint/blob/main/doc/md025.md) you should be able to disable checking for a frontmatter title by setting the `front_matter_title` setting to `""`.

> To disable the use of front matter by this rule, specify "" for front_matter_title.

However, this doesn't seem to work and MD025 violations will not be reported if `front_matter_title` is set to `""` and a frontmatter title exists. 

```shell
$ npm ls -a | grep markdownlint
├─┬ markdownlint-cli@0.44.0
│ ├─┬ markdownlint@0.37.4

$ grep -A2 MD025 .markdownlint.yaml 
# MD025/single-title/single-h1 - Multiple top-level headings
MD025:
  # Disable checking of YAML frontmatter.
  front_matter_title: ""

$ cat <<EOF > test.md
> ---
> title: Test
> ---
> 
> No h1 heading.
> EOF

$ npx markdownlint test.md && echo "no errors"
no errors
```

**Comment by ianlewis** [comment]
Oops. I got this confused with MD041
